electron/src/lib/CoreProtocol.ts

```diff
@@ -93,26 +93,50 @@ export class CustomProtocolHandler {
       app.setAsDefaultProtocolClient(config.customProtocolName);
     }
 
+    ipcMain.on(EVENT_TYPE.ACTION.DEEP_LINK_SUBMIT, async (_event, url: string) => {
+      try {
+        await this.dispatchDeepLink(url);
+      } catch (error) {
+        logger.error(error);
+      }
+    });
+
     if (platform.IS_MAC_OS) {
       app.on('open-url', async (event, url) => {
         event.preventDefault();
-        await this.dispatchDeepLink(url);
+        try {
+          await this.dispatchDeepLink(url);
+        } catch (error) {
+          logger.error(error);
+        }
       });
     } else {
       app.once('ready', async () => {
         logger.info('App ready, looking for deep link in arguments ...');
         const deepLink = this.findDeepLink(process.argv);
         if (deepLink) {
-          logger.info('App ready, dispatching deep link ...');
-          await this.dispatchDeepLink(deepLink);
+          logger.info('Dispatching deep link ...');
```

Comment:
```suggestion
          logger.info('Dispatching deep link...');
```

Reply:
We are using a space before ellipses in all the other places, too. I am going to keep the space for consistency.